src/test/utils.js

```diff
@@ -41,3 +42,29 @@ export const runAuthenticatedQuery = (query, rootValue = {}) => {
     Object.assign({ accessToken: "secret", userID: "user-42" }, rootValue)
   )
 }
+
+let mergedSchema
+
+/**
+ * Same as `runQuery`, but runs against stitched schema
+ *
+ * @param {String} query      The GraphQL query to run.
+ * @param {Object} rootValue  The request params, which currently are `accessToken` and `userID`.
+ * @see runQuery
+ */
+export const runQueryMerged = async (
+  query,
+  rootValue = { accessToken: null, userID: null }
+) => {
+  if (!mergedSchema) {
+    mergedSchema = await mergeSchemas()
+  }
+  return graphql(mergedSchema, query, rootValue, {}).then(result => {
+    if (result.errors) {
+      const error = result.errors[0]
+      throw error.originalError || error
+    } else {
+      return Promise.resolve(result.data)
```

Comment:
Off-topic: I’m wondering why we’re boxing the result data in a resolved promise here and above, that should be done automatically when returning data from a `then`, no?

Reply:
Indeed it's superficial. I just copied :)